templates/auction/index.tmpl

```diff
@@ -116,11 +131,15 @@
         url: `/login`,
         method: 'POST',
         success: function (result) {
-          $("#login-status-message").html("Successfully logged in");
+          M.toast({
+            html: "Successfully logged in"
+          });
```

Comment:
if reload later, then user will not see this toast.
you can use alert(string), that will show before page reloading.

Reply:
Or, you can use
```
setTimeout(function () {
  location.reload();
}, 2000);
```

which enables it refresh after two seconds, so that user can see the toast.